UNUMColorPicker/Classes/UNUMColorPickerViewModel.swift

```diff
@@ -11,7 +11,7 @@ import Foundation
 class UNUMColorPickerViewModel {
 
     let colors: [UIColor]
-    let initialColor: UIColor
+    var initialColor: UIColor
```

Comment:
why does this need to be a `var`? It's set in the initializer; when would it change?